### PR TITLE
[Ubuntu] update zstd version

### PR DESCRIPTION
**Pull request by Alexey-Ayupov**
# Description
In order to get the latest zstd version we had to change the way of getting the app.

The current approach works with the official facebook repository in GitHub.

#### Related issue:
https://github.com/actions/runner-images-internal/issues/4404

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/linux/scripts/installers/zstd.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+apt-get install -y liblz4-dev
+release_tag=$(curl https://api.github.com/repos/facebook/zstd/releases/latest | jq -r '.tag_name')
+zstd_tar_name="zstd-${release_tag//v}.tar.gz"
+$URL="https://github.com/facebook/zstd/releases/download/${release_tag}/${zstd_tar_name}"
+download_with_retries "${URL}" "/tmp" "${zstd_tar_name}"
+tar xzf /tmp/$zstd_tar_name -C /tmp/
+cd "/tmp/zstd-${release_tag//v}"
```
Comment (mikhailkoliada) [inline images/linux/scripts/installers/zstd.sh:16]: nitpick: no need in quotes

File: images/linux/scripts/installers/zstd.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+make -C contrib/pzstd all
+make install
+cp "/tmp/zstd-${release_tag//v}/contrib/pzstd/pzstd" /usr/local/bin/
```
Comment (mikhailkoliada) [inline images/linux/scripts/installers/zstd.sh:19]: same

File: images/linux/scripts/installers/zstd.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+rm -rf "/tmp/zstd-${release_tag//v}"
```
Comment (mikhailkoliada) [inline images/linux/scripts/installers/zstd.sh:20]: same

File: images/linux/scripts/installers/zstd.sh (hunk 1 above)
Comment (Alexey-Ayupov) [reply images/linux/scripts/installers/zstd.sh]: quotes were removed

File: images/linux/scripts/installers/zstd.sh (hunk 2 above)
Comment (Alexey-Ayupov) [reply images/linux/scripts/installers/zstd.sh]: quotes were removed

File: images/linux/scripts/installers/zstd.sh (hunk 3 above)
Comment (Alexey-Ayupov) [reply images/linux/scripts/installers/zstd.sh]: quotes were removed

**Review by mikhailkoliada — APPROVED** [review]

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**